src/theme.cpp

```diff
@@ -83,6 +107,9 @@ DecTriple::DecTriple()
     for (auto it : children) {
         it->scheme_changed_.connect([this]() { this->triple_changed_.emit(); });
     }
+    active.setChildDoc("configures the decoration the focused client");
+    normal.setChildDoc("configures the decoration of focused clients");
```

Comment:
Is it really the focused client?

Reply:
Fixed.

Reply:
I know they make things clearer, but are the negatives really needed? Imagine you decide to add another special decoration; do you want to add another word here? Or otherwise: is the condition *explicitly* that the client is unfocused & non-urgent, or is it the fallback?

Sorry for being so perfectionist, I cannot help myself :(

Reply:
I didn't know a better way to express it and didn't just want to say 'normal' here. But I think I now have a better doc: "the default decoration scheme for clients"

Reply:
I like the new description much better, thank you!

Reply:
Thanks!